simulation-runner/config/scenarios/ApplesToApples.json

```diff
@@ -1,5 +1,5 @@
 {
-    "run_description": "'role::by_size_large' role from perf control repo, 750 agents, 8 iterations",
+    "run_description": "'role::by_size_large' role from perf control repo, 600 agents, 8 iterations",
```

Comment:
why was this changed in this PR?

Reply:
Probably should have been a different PR, just noticed this was wrong and fixed it.

Reply:
in the future that might be better.  but in the meantime, especially with "magic numbers" this might benefit from a commit statement about why it was wrong and why it was changed to what it was changed... to

Reply:
👍